### DISCUSSION TITLE
Deconvolute with own reference matrix

### DISCUSSION THREAD

**Discussion by FranziskaGoertler** [omnideconv/immunedeconv#143] [Q&A]
How can I use my own reference matrix for deconvolution? Like in CIBERSORT it is possible do use the own reference matrix. Using predefined reference matrices is not fitting to the cell types in my data and the results are always different cell types for all methods.

**Comment by LorenzoMerotto** [comment]
Hello!
The usage of custom signature matrices is not allowed by all methods. 
Currently, `CIBERSORT`, `EPIC`, `BASE` and `ConsensusTME` are the only one that you can use. You can access them via the `deconvolute_methd_custom` functions

**Reply by FranziskaGoertler** [reply]
Thank you, found it.
